tools/iwyu/glmimp.py

```diff
@@ -0,0 +1,57 @@
+# generate glm include what you use mapping
+
+import json
+
+from pathlib import Path
+
+
+def begins_with(alist, prefix):
+    if len(prefix) > len(alist):
+        return False
+    return alist[:len(prefix)] == prefix
+
+
+glm_base = "/Users/peter/Work/Inviwo/inviwo/ext/glm/glm"
```

Comment:
Is this a generated file? Otherwise, this path should probably be relative.

Reply:
Yes, but you only have to do it ones kind of, or of glm does some large changes...